### PR TITLE
Added Remove-AllContainers function

### PR DESCRIPTION
**Pull request by katrybacka**

### CODE REVIEW

File: CIScripts/Test/RemoveAllContainers.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        } | Should BeNullOrEmpty
+    }
+
+    It "Removes mutliple containers if exist" {
+        New-Container -Session $Session -NetworkName $NetworkName
+        New-Container -Session $Session -NetworkName $NetworkName
+        New-Container -Session $Session -NetworkName $NetworkName
+        Invoke-Command -Session $Session -ScriptBlock {
+            docker ps -aq
+        } | Should Not BeNullOrEmpty
```
Comment (jacek-izykowski) [inline CIScripts/Test/RemoveAllContainers.Tests.ps1]: Possible refinement: check if there are actually multiple containers.

**Comment by katrybacka** [comment]
recheck no bug

**Comment by katrybacka** [comment]
recheck no bug

**Comment by katrybacka** [comment]
recheck no bug

**Pull request merged**